site/phpstan-baseline.neon

```diff
@@ -11286,6 +11286,11 @@ parameters:
 			count: 1
 			path: app/models/gradeable/GradeableUtils.php
 
+		-
```

Comment:
We want to avoid adding to the baseline if possible. Looking at the code, it seems to be because you put the definitions for the consts between the declaration of getGradeablesFromCourse and the doc string above it. Move the const definitions and this should go away.